internal/tiltfile/tiltfile_state.go

```diff
@@ -655,7 +655,7 @@ func (s *tiltfileState) assembleK8sV2() error {
 			}
 
 			for i, o := range opts.objects {
-				entities, ok := fragmentsToEntities[o]
```

Comment:
i'm surprised they're not stored in objects as lowercase?

Reply:
Hmm it might be weird to use a different string than the one the user entered in error messages though, even if we treat it as case insensitive. I think I want to preserve the original string.